test-alerts.cpp

```diff
@@ -62,47 +66,42 @@ TEST_CASE("infers the breach according to limits based on cooling type ") {
 		double temperatureInC;
 	}test_InputStructure;
 
-	int const NoofTestCases = 3;
-	test_InputStructure  test_InputMatrix[NoofTestCases] = { {TO_CONSOLE  , { PASSIVE_COOLING ,  ""  }, 12},
-								 {TO_CONTROLLER,{ HI_ACTIVE_COOLING, ""  }, 12},
-								 {TO_EMAIL     ,{ MED_ACTIVE_COOLING, "" }, 12},
-	};
-	double test_OutputMatrix[NoofTestCases] = {  { SUCCESS },
-		                                     { SUCCESS },
-						     { SUCCESS },
+	
+	int const NoofTestCases = 5;
+	test_InputStructure  test_InputMatrix[NoofTestCases] = { {TO_CONTROLLER, { PASSIVE_COOLING ,  ""  }, 12},
```

Comment:
Why not keep test input matrix and output matrix in the same structure? Then we avoid the mistake that someone will add or remove an input without doing the same in the output...